app/grandchallenge/algorithms/templates/algorithms/algorithmpermissionrequest_form.html

```diff
@@ -3,6 +3,9 @@
 {% load clean from bleach %}
 {% load guardian_tags %}
 
+{% block title %}
+    {% if "change_algorithm" in algorithm_perms %}Review access request{% else %}Request access{% endif %} - {{ algorithm.title }} - {{ block.super }}
```

Comment:
Not sure if this works. the `get_obj_perms` are block scoped. As such the `algorithm_perms` will likely not exist here?

Reply:
Thanks for mentioning this. I did not know that. I will double check.

Reply:
I assume it checks out? =)